src/psyclone/tests/test_files/dynamo0p3/testkern_two_int_scalars.f90

```diff
@@ -6,7 +6,12 @@
 !-------------------------------------------------------------------------------
 ! Author R. Ford STFC Daresbury Lab
 
-module testkern
+module testkern_two_int_scalars
```

Comment:
Please add yourself to the author list and update the copyright notice/date if necessary.

Reply:
Added myself as an author, but which copyright notice?